### ISSUE TITLE
Encrypted icon status is too subtle

### ISSUE THREAD

**Issue by keverets** [dino/dino#971]
I had an OMEMO encrypted chat between myself and a contact that was performed using Conversations on Android. Then I moved to dino on the laptop to continue the conversation. Previous lines all showed locked icons and were readable, indicating that OMEMO was working as expected.

I entered a message and sent it, and then immediately (but too late) saw a red unlocked icon indicating that the message was sent unencrypted.

The grey unlocked icon to the right of the message did not sufficiently indicate that the message was going to be sent unencrypted until after the message is sent.

Please consider making the "unlocked" icon for sending more distinct, using red or similar to what is done for the message post-sending, to avoid user error resulting in messages being sent unsecured. Using a mechanism similar to Conversations that places hint text in the text field of "Send OMEMO encrypted message" or "Send unencrypted message" would also be beneficial for those with some form of colourblindness.

A default of continuing a conversation using OMEMO if the previous lines had been similarly encrypted would also improve security.

Thanks for making dino, it is generally a very nice and easy to use XMPP client. Just a little too easy for people to accidentally expose themselves at the moment ;)

**Comment by keverets** [comment]
Images of the icons as presented in debian, 0.2.0-1~bpo10+1 (from buster-backports).

Lots of lines with: ![dino-message-sent-encrypted](https://user-images.githubusercontent.com/502780/103238500-ee3dc380-4918-11eb-95f5-6e8b811b91ad.png)

Didn't notice that this: ![dino-unlocked-message-send](https://user-images.githubusercontent.com/502780/103238417-a5860a80-4918-11eb-9b52-3cef21ac331b.png)

was not this: ![dino-locked-message-send](https://user-images.githubusercontent.com/502780/103238416-a4ed7400-4918-11eb-987d-9e8d787669f8.png)

Until the message line included: ![dino-message-sent-unencrypted](https://user-images.githubusercontent.com/502780/103238414-a4ed7400-4918-11eb-99d6-1a08c270484b.png)

**Comment by licaon-kter** [comment]
Covered maybe by "omemo by default"

**Comment by keverets** [comment]
> Covered maybe by "omemo by default"

That would definitely help avoid the initial problem. Though it does leave the issue of an unsecured connection not immediately and obviously indicated. Most browsers have improved on this in the past: used to have the same problem for http vs https; difficult to see a single character difference and the lock icon had the same subtle differences.

Now Firefox for instance does:  ![firefox-unsecure](https://user-images.githubusercontent.com/502780/103238935-2eea0c80-491a-11eb-90df-a78c1a52a8c5.png)

And Chromium does: ![chromium-not-secure](https://user-images.githubusercontent.com/502780/103238962-3b6e6500-491a-11eb-8301-017c28846ae2.png)

**Comment by jelbana** [comment]
This happens to me too. I occasionally log out and re-login to my account and then don't notice until after I've sent a message that the previous conversation I had that was encrypted is now unsecured. The default not being unsecured would help but I agree the icon is not quite eye-catching.

**Comment by mmigel** [comment]
I agree. The padlock in the message entry window could be green or red depending on the setting, or something similar. I also sometimes send a message with disabled encryption.

**Comment by kousu** [comment]
Conversations colours the background of messages by encryption:

![signal-2021-02-15-132347](https://user-images.githubusercontent.com/987487/107982075-63c92600-6f91-11eb-80e0-c2111ac991b2.jpeg)

I think this is really good!

I also *stops* colouring the background if you turn off encryption. I don't really like that but I haven't brainstormed a better way to handle that.

**Comment by eerielili** [comment]
@kousu
Yeah, I like the background color thing, it also can play nicer with themes. I made a branch where it's greenish when encryption is on :
![image](https://github.com/dino/dino/assets/81772782/9a17d33b-588a-436f-af02-a853f212eff6)

And yellow (I prefer it than red, I think red implies errors more than warning), when encryption is turned off:
![image](https://github.com/dino/dino/assets/81772782/01e5737b-bbdf-42f6-884d-5d5d1002cdc9)
